File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/ifpi-ads-2023.2/modulo4_condicionais/bolsa_ifpi.js
import { question } from "readline-sync";

function main(){
    const renda_familiar = Number(question('Renda Familiar: '))

    const valor_auxilio = calcular_bolsa(renda_familiar)

    print(`Valor da Bolsa é: R$ ${valor_auxilio.toFixed(2)}   `)
}

function calcular_bolsa(renda_familiar){
    /*
     * 0 até 1SM --> 800
     * 1SM+ - 2SM --> 600
     * 2SM  - 3SM --> 400
     * 3SM+       --> 200
     */
    const salario_minimo = 1320
    if (renda_familiar <= salario_minimo){
        return 800
    }else if(renda_familiar <= 2 * salario_minimo){
        return 600
    }else{
        return 200
    }

}

    function print(label){
        console.log(`${label}`)
    }



main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/ifpi-ads-2023.2/modulo4_condicionais/bolsa_ifpi.py
def main():
    renda_familiar = float(input('Renda Familiar: '))
    valor_auxilio = calcular_bolsa(renda_familiar)
    print(f'O valor da bolsa é de: {valor_auxilio}'.format(round(valor_auxilio, 2)))


def calcular_bolsa(renda_familiar):
    # 0 até 1SM --> 800
    # 1SM+ - 2SM --> 600
    # 2SM+ - 3SM --> 400
    # 3SM+       --> 200
    salario_minino = 1320
    if renda_familiar <= salario_minino:
        return 800
    elif renda_familiar <= 2 * salario_minino:
        return 600
    elif renda_familiar <= 3 * salario_minino:
        return 400
    else:
        return 200
    

main()